react/components/molecules/keyboard/keyboard.js

```diff
@@ -39,13 +41,52 @@ export class Keyboard extends Component {
         super(props);
 
         this.state = {
-            symbolicKeyboard: this.props.symbolicKeyboard
+            symbolicKeyboard: this.props.symbolicKeyboard,
+            animationPositionY: new Animated.Value(212),
+            animationOpacity: new Animated.Value(0)
         };
 
         this.keys = "1234567890qwertyuiopasdfghjklzxcvbnm";
         this.keyColumns = [10, 10, 9, 9];
     }
 
+    componentDidMount() {
+        this.showKeyboardAnimated();
+    }
+
+    showKeyboardAnimated = async () => {
+        if (!this.props.animations) return;
+
+        await this._animateKeyboard(0, 1);
+    };
+
+    hideKeyboardAnimated = async () => {
+        if (!this.props.animations) return;
+
+        await this._animateKeyboard(212, 0, 200);

```

Comment:
why 200 here?

Reply:
Also, please move 212 to a variable like `this.keyboardPositionY`

Reply:
200 because on close subjectively I thought it felt better using a short animation

Reply:
adding as an argument `animationDuration`